registry/registry_db.go

```diff
@@ -270,18 +354,139 @@ func dbGetPrefixJwks(prefix string, approvalRequired bool) (*jwk.Set, error) {
 	return &set, nil
 }
 
+func getNamespaceById(id int) (*Namespace, error) {
+	if id < 1 {
+		return nil, errors.New("Invalid id. id must be a positive number")
```

Comment:
Side-note: this check is reasonable for a SQLite backend.  In the future, we may have alternate ID formats.

It's OK to leave for now but would be good to have a mental picture of the ID being somewhat arbitrary.